tests/testthat/test-fimsframe.R

```diff
@@ -11,9 +11,18 @@ test_that("Can create the S4 FIMSFrame classes", {
   expect_vector(age_frame@ages, ptype = integer())
   expect_s3_class(age_frame@weightatage, "data.frame")
   expect_s3_class(fims_frame@data, "data.frame")
-  expect_silent(plot(fims_frame))
-  expect_silent(plot(age_frame))
-  dev.off()
+
+  # A helper function that creates a figure from code
+  save_png <- function(code, width = 400, height = 400) {
+    path <- tempfile(fileext = ".png")
+    png(path, width = width, height = height)
+    on.exit(dev.off())
```

Comment:
@Bai-Li-NOAA do you care that the png is not deleted?

Reply:
I was trying to make sure that `dev.off()` is called before exiting the function and before deleting the temporary files. Maybe I am overthinking things 😄